scripts/package/deb-after-install.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
```

Comment:
Really what's happening here is that we're using FPM instead of the fundamental packaging tools.  This sort of stuff can (and does) occur in packaging all the time but we're not used to seeing it because we don't often examine source packages.  Given that, a cross-distro post-config script would - from a package perspective - be on the janky side; of course, we're already using FPM, which is pretty strong declaration that we're not dedicated to package purity in the first place. :smirk:

Ultimately, either we put this stuff into an omnibus post-install script, or we leave it in the package-specific scripts that FPM expects.  Either way we need to maintain a bunch of shell code, except that in the former case, it's shell code that everybody *expects*, which seems better to me.